### DISCUSSION TITLE
Call for Linux/Windows maintainers

### DISCUSSION THREAD

**Discussion by epwalsh** [epwalsh/obsidian.nvim#274] [Announcements]
I'm looking for Linux and Windows users who are confident in Lua to help develop and maintain this plugin, since currently I'm only developing on a Mac. I would occasionally ping you to help review PRs or fix bugs specific to your operating system, when you have time.

If you're interested please comment here or send me a email with a couple sentences describing your experience with Lua and Neovim plugin development.

**Comment by sQVe** [comment]
👋,

I would be happy to help with Linux-land. I started building a second brain in Obsidian a week ago, and your plugin has been the missing 🧩 in my note taking. 

I've already found a couple of issues, which I will look into in the near future (Christmas vacation and all).

I'm well versed in Lua and Neovim.

**Reply by epwalsh** [reply]
@sQVe great! I appreciate that!
